### PR TITLE
New Dns debug endpoint

### PR DESCRIPTION
**Pull request by ldemailly**
```bash
$ curl -s localhost:8080/fortio/rest/dns?name=debug.fortio.org | jq
```
Returns
```json
{
  "Name": "debug.fortio.org",
  "IPv4": [
    "18.222.136.83",
    "192.9.142.5",
    "192.9.227.83"
  ],
  "IPv6": [
    "2600:1f16:9c6:b400:282c:a766:6cab:4e82",
    "2603:c024:c00a:d144:7cd0:4951:7106:96b8",
    "2603:c024:c00a:d144:6663:5896:7efb:fbf3"
  ]
}
```

Also added as a form in the main UI page

For review, only look at https://github.com/fortio/fortio/commit/6db71e047498add38f7e188efbc52f0265e7ddda

### CODE REVIEW

**Review by wuhaoyujerry — APPROVED** [review]

**Review by bseenu — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -406,7 +408,8 @@ Fortio X.Y.Z grpc 'ping' server listening on tcp [::]:8079
 Fortio X.Y.Z https redirector server listening on tcp [::]:8081
 Fortio X.Y.Z http-echo server listening on tcp [::]:8080
 Data directory is /Users/ldemailly/dev/fortio
-REST API on /fortio/rest/run, /fortio/rest/status, /fortio/rest/stop
+REST API on /fortio/rest/run, /fortio/rest/status, /fortio/rest/stop, /fortio/rest/dns
+Debug endpoint on /debug, Additional Echo on /debug/echo/, flags on /fortio/flags, and Metrics endpoint on /debug/metrics
```
Comment (wuhaoyujerry) [inline README.md]: I would capitalized `flags` here to keep it consistent, but it's just my OCD

File: rapi/restHandler.go [hunk 2]
```diff
@@ -531,6 +539,31 @@ func RemoveRun(id int64) {
 	log.LogVf("REST Removed run %d", id)
 }
 
+func RESTDNSHandler(w http.ResponseWriter, r *http.Request) {
+	log.LogRequest(r, "REST DNS Api call")
```
Comment (wuhaoyujerry) [inline rapi/restHandler.go]: I would make capitalize API here

Comment (ldemailly) [reply rapi/restHandler.go]: done (kinda, removed api actually, it was also in other lines)

File: README.md (hunk 1 above)
Comment (ldemailly) [reply README.md]: done see update

File: fnet/network.go [hunk 3]
```diff
@@ -421,6 +407,31 @@ func ResolveByProto(ctx context.Context, host string, port string, proto string)
 	return dest, nil
 }
 
+// ResolveAll returns all the IPs for the host.
+// resolveType is a filter for the IPs to use, `ip4` for ipv4, `ip6` for ipv6, or `ip` or "" for both.
```
Comment (ldemailly) [inline fnet/network.go:411]: Note for posterity: this came from multicurl which came from here...

**Pull request merged**